### DISCUSSION TITLE
Blank screen while booting Windows 11 Arm on iOS platform with synchronous exception at 0x7FEEABDB

### DISCUSSION THREAD

**Discussion by scaraOvO** [utmapp/UTM#6915] [UTM Help (iOS)]
Nothing will be displayed from the moment I turn on the virtual machine (including the UEFI loading screen. When I switch to Serial output and then restart, I will see Synchronous exception at 0x7FEEABDB

UTM 4.6.3
iOS 15.5
iPad Pro 2018

**Comment by KingIsMe707** [comment]
Don’t worry it’s not problem just wait
